### PR TITLE
Changes to the Customer Zone UI

### PR DESCRIPTION
**Pull request by mscodepl**
Changes to the Customer Zone UI (Invoices to be paid, list of active services)

image

### CODE REVIEW

**Comment by mscodepl** [comment]
The button to pay is as TODO, need to add a redirect to it straight to the invoice for this products

**Comment by mscodepl** [comment]
image
image
image

**Comment by mscodepl** [comment]
image

**Comment by mscodepl** [comment]
image

**Comment by mscodepl** [comment]
+ Added invoices to the admin menu
image

image

**Comment by mscodepl** [comment]
**TODO**
- [ ] Transfer showTicketDate function to Ticket

File: themes/default/views/admin/tickets/show.blade.php [hunk 1]
```diff
@@ -66,56 +75,101 @@ class="mt-2 w-full">
...
+
                 </div>
             </div>
         </div>
     @endif
     <br>
-    <div class="overflow-hidden bg-white sm:rounded-lg dark:bg-secondary-100">
-        <form method="POST" action="{{ route('admin.tickets.reply', $ticket->id) }}" class="mt-10">
+    <div class="overflow-hidden bg-white shadow-xl sm:rounded-lg dark:bg-secondary-100">
+        <form method="POST" action="{{ route('clients.tickets.reply', $ticket->id) }}" class="mt-10" id="reply">
```
Comment (CorwinDev) [inline themes/default/views/admin/tickets/show.blade.php:136]: This should be admin.tickets.reply and not clients.tickets.reply

**Review by CorwinDev — CHANGES_REQUESTED** [review]

**Review by CorwinDev — APPROVED** [review]

**Pull request merged**